specs/ReleaseChannelPreferenceSet.md

```diff
@@ -0,0 +1,325 @@
+            // `RuntimeChannelPreferenceSet` can be overridden by the corresponding registry
+            // override `RuntimeChannelPreferenceSet` or the environment variable
+            // `WEBVIEW2_RUNTIME_CHANNEL_PREFERENCE_SET`. Set the value to a comma-separated string of
+            // integers, which map to the `CoreWebView2RuntimeChannel` values:
+            // Stable (0), Beta (1), Dev (2), and Canary (3). For example, the value "0,2"
+            // indicates that the loader should only search for Dev channel and the WebView2
+            // Runtime, using the order indicated by `ShouldReverseChannelSearchOrder`.
+            // The loader attempts to interpret each integer and treats any invalid entry as
+            // Stable channel.
+            CoreWebView2RuntimeChannel RuntimeChannelPreferenceSet { get; set; };
+
+            // The `ShouldReverseChannelSearchOrder` property is `false` by default and
+            // the WebView2 loader searches for a Runtime channel on the machine from
+            // most to least stable using the first channel found. The default search order is:
+            // WebView2 Runtime -> Beta -> Dev -> Canary. Set `ShouldReverseChannelSearchOrder`
+            // to `true` to reverse the search order so that the loader searches for a channel
+            // from least to most stable. If a `RuntimeChannelPreferenceSet` has been provided, the
+            // loader will only search for channels in the set. See `CoreWebView2RuntimeChannel`
+            // for more details on channels. This property can be overridden by the corresponding
+            // registry key `ReverseChannelSearchOrder` or the environment variable
+            // `WEBVIEW2_REVERSE_CHANNEL_SEARCH_ORDER`. Set the value to `1` to reverse the search
+            // order. See `CreateCoreWebView2EnvironmentWithOptions` for more details on
+            // overrides.
+            Boolean ShouldReverseChannelSearchOrder { get; set; };
+        }
+    }
+
+    runtimeclass CoreWebView2Environment
+    {
+        // ...
+
+        [static_name("Microsoft.Web.WebView2.Core.ICoreWebView2EnvironmentStatics")]
+        {
+            // ...
+
+            // Get the browser version info including channel name if it is not the
+            // WebView2 Runtime.  Channel names are Beta, Dev, and Canary.
+            // If an override exists for `BrowserExecutableFolder`, `RuntimeChannelPreferenceSet`,
+            // or `ReverseChannelSearchOrder`, the override is used.  `BrowserExecutableFolder`
+            // takes precedence over the other options. See `CreateCoreWebView2EnvironmentWithOptions`
+            // for more details on overrides. If an override is not specified, then the
+            // parameters passed to `GetAvailableCoreWebView2BrowserVersionStringWithOptions`
+            // are used. The method fails if the loader is unable to find an installed WebView2
```

Comment:
>The method fails if...

What is the nature of the failure? Does it throw an exception?

Reply:
Agreed that this isn't the best but we match the existing same named method's (of which this is an overload) error behavior of throwing an exception. (No changed)